### PR TITLE
ui: Tune and clean up ORT Run results page

### PR DESCRIPTION
**Pull request by Etsija**
Please see the commits for details.

Screenshot from 2024-12-11 14-27-43

Screenshot from 2024-12-11 14-34-44

Screenshot from 2024-12-11 14-38-15

Screenshot from 2024-12-11 14-39-44

### CODE REVIEW

File: ui/src/routes/organizations/$orgId/products/$productId/repositories/$repoId/runs/$runIndex/-components/rule-violations-statistics-card.tsx [hunk 1]
```diff
@@ -39,7 +43,7 @@ export const RuleViolationsStatisticsCard = ({
   const { data, isPending, isError, error } =
     useRuleViolationsServiceGetRuleViolationsByRunId({
       runId: runId,
-      limit: 1,
+      limit: 100000,
```
[the remark this answers is not in the document]
Comment (Etsija) [reply ui/src/routes/organizations/$orgId/products/$productId/repositories/$repoId/runs/$runIndex/-components/rule-violations-statistics-card.tsx]: As a similar magic number was used throughout the UI code (also in files not related to this PR) for the same purpose, I decided to clean the code with an additional commit that touches all of them.

File: ui/src/routes/organizations/$orgId/products/$productId/repositories/$repoId/runs/$runIndex/-components/packages-statistics-card.tsx [hunk 2]
```diff
@@ -89,21 +76,39 @@ export const PackagesStatisticsCard = ({
 
   const packagesTotal = data.pagination.totalCount;
 
+  const value = jobIncluded
+    ? status === undefined
+      ? '-'
+      : !['FINISHED', 'FINISHED_WITH_ISSUES', 'FAILED'].includes(status)
```
Comment (mmurto) [inline ui/src/routes/organizations/$orgId/products/$productId/repositories/$repoId/runs/$runIndex/-components/packages-statistics-card.tsx:82]: See previous.

File: ui/src/routes/organizations/$orgId/products/$productId/repositories/$repoId/runs/$runIndex/-components/rule-violations-statistics-card.tsx [hunk 3]
```diff
@@ -67,14 +74,40 @@ export const RuleViolationsStatisticsCard = ({
     return;
   }
 
-  const ruleViolationsTotal = data.pagination.totalCount;
+  const violationsTotal = data.pagination.totalCount;
+
+  const value = jobIncluded
+    ? status === undefined
+      ? '-'
+      : !['FINISHED', 'FINISHED_WITH_ISSUES', 'FAILED'].includes(status)
```
Comment (mmurto) [inline ui/src/routes/organizations/$orgId/products/$productId/repositories/$repoId/runs/$runIndex/-components/rule-violations-statistics-card.tsx:82]: See previous.

Comment (Etsija) [reply ui/src/routes/organizations/$orgId/products/$productId/repositories/$repoId/runs/$runIndex/-components/rule-violations-statistics-card.tsx:82]: @mmurto Once again, fearless rebasing fails me miserably, so I'll either do a separate fixup commit on top of my PR, or waste another day by rebasing on raw Git.

**Review by mmurto — APPROVED** [review]

**Pull request merged**